.github/workflows/reusable-operator-hub-release.yaml

```diff
@@ -0,0 +1,91 @@
+name: Reusable - Create operator hub pull request
+
+on:
+  workflow_call:
+    inputs:
+      org:
+        type: string
+        required: true
+      repo:
+        type: string
+        required: true
+
+jobs:
+  create-operator-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+      - name: Set version as env variable
+        env:
+          TAG: ${{ github.ref_name }}
+        run: |
+          echo $TAG
+          TAG=${TAG:1} # remove v (prefix)
+          echo version=${TAG} >> $GITHUB_ENV # update GitHub ENV vars
+
+      - name: Sync fork
+        env:
+          GH_TOKEN: ${{ steps.generate_token.outputs.token }}
+        run: |
+          # synchronizing the fork is fast, and avoids the need to fetch the full upstream repo
+          # (fetching the upstream repo with "--depth 1" would lead to "shallow update not allowed"
+          #  error when pushing back to the origin repo)
+          gh repo sync grafana/${{ inputs.repo }} \
+              --source ${{ inputs.org }}/${{ inputs.repo }} \
+              --force
+
+      - name: Checkout operatorhub repo
+        uses: actions/checkout@v3
+        with:
+          repository: grafana/${{ inputs.repo }}
```

Comment:
this looks fine

https://github.com/grafana/community-operators
https://github.com/grafana/community-operators-prod